Linkta-org/core
# feat(#385): Consistent yellow pill buttons

## Updates all yellow pill buttons to match Log In / Log Out buttons (that is, "secondary-main" color normally, and "secondary-dark" on hover, and text color "secondary" regardless of hover state).

Partially Fixes #385 

## Type of change

🐛 Bug fix (non-breaking change which fixes an issue)

Sign In / Sign Up / Save
![Screenshot from 2024-09-26 20-25-33](https://github.com/user-attachments/assets/4cb285fa-2d2b-42a8-aadd-f92e39388843)

404 page
![Screenshot from 2024-09-26 20-25-45](https://github.com/user-attachments/assets/a67504d3-9e49-4d8f-8695-1b32090837dc)

Error Page
![Screenshot from 2024-09-26 20-28-16](https://github.com/user-attachments/assets/9d764b2c-1a5d-4f2f-aa7f-7cdb92affe89)

Sign In Page
![Screenshot from 2024-09-26 20-35-44](https://github.com/user-attachments/assets/bfd069c2-6b9c-4e57-88c9-c7ef6e62e611)

Create Account Page
![Screenshot from 2024-09-26 20-46-05](https://github.com/user-attachments/assets/a7724803-ab85-4368-b278-712acb4334f0)

Send Link Page
![Screenshot from 2024-09-26 20-38-35](https://github.com/user-attachments/assets/c1202c37-c152-4134-8a8b-da9f3c178dfb)

Generate
![Screenshot from 2024-09-26 20-47-10](https://github.com/user-attachments/assets/38b71aa2-58e4-45cb-9218-89d5bb1efcfb)

Linkta flow confirmation dialog:
![Screenshot from 2024-09-30 21-46-57](https://github.com/user-attachments/assets/5991b7b8-5644-419c-a150-a7605b3be1fb)

## How Can this be tested? Testing Plan to review this PR

User should review the above pages and ensure:
1. Medium yellow color normally
2. Darker color on hover
